snuba/state/cache/redis/backend.py

```diff
@@ -224,6 +232,8 @@ def build_notify_queue_key(task_ident: str) -> str:
                 task_ident,
                 effective_timeout,
             )
+            queue_len = self.__client.llen(build_notify_queue_key(task_ident))
+            metrics.gauge("notify_queue_len", queue_len, tags=metric_tags)
```

Comment:
Are you sure you want a gauge here? Every hit to this metric will overwrite the previous value. Unless the `metrics_tags` provide enough granularity for what you want to track.

Reply:
hmm I guess what I wanted is a distribution of the queue lengths. Is that not possible with datadog?

Reply:
Ah I need to use the histogram type

https://docs.datadoghq.com/metrics/types/?tab=histogram#metric-types

Our metrics abstraction doesn't support this yet so maybe I'll take it out for now and put it back in later